components/playlist/common/features.cc

```diff
@@ -12,4 +12,8 @@ namespace playlist::features {
 
 BASE_FEATURE(kPlaylist, "Playlist", base::FEATURE_DISABLED_BY_DEFAULT);
 
+BASE_FEATURE(kPlaylistFakeUA,
+             "Playlist Fake UA",
```

Comment:
a feature name with spaces? 🤨

Reply:
Ah yeah, I was confused with about_flags. This should be dash(-)ed, right? So that we can use it through commandline too?

Reply:
> Ah yeah, I was confused with about_flags. This should be dash(-)ed, right? So that we can use it through commandline too?

no dashes, no spaces, just `CamelCase`.